bin/fin

```diff
@@ -5208,6 +5208,12 @@ load_configuration ()
 	fi
 	export DOCROOT=${DOCROOT:-docroot}
 
+	if [[ $(echo "${VIRTUAL_HOST}" | grep -c -v "^[a-z0-9\-\.]*$") -gt 0 ]]; then
```

Comment:
I have a refactoring proposition

```bash
local OLD_VHOST=${VIRTUAL_HOST}
export VIRTUAL_HOST=$(echo ${VIRTUAL_HOST} | tr '[:upper:]' '[:lower:]' | sed 's/_/-/g' | sed 's/[^-a-z0-9\.]//g')
# Warn in case Virtual Host was auto-corrected
if [[ "$OLD_VHOST" != "$VIRTUAL_HOST" ]]; then
    echo-warning "The ${yellow_bold}VIRTUAL_HOST${yellow} has been modified from ${yellow_bold}${OLD_VHOST}${yellow} to ${yellow_bold}${VIRTUAL_HOST}${NC} ${yellow}to comply with browser standards."
fi
```